### PR TITLE
Add PrideToken (PRD)

### PR DESCRIPTION
**Pull request by senegalbgc**
This pull request adds the PrideToken (PRD) details to the Celo token list.  
  
**Token Symbol:** PRD    
**Token Name:** PrideToken    
**Contract Address:** 0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd    
**Decimals:** 18    
  
**Verified Across Multiple Blockchains:**  
- **Ethereum:** Verified on Etherscan.  
- **Celo:** Verified on Celo Explorer via Sourcify.  
- **BSC:** Verified on BscScan.  
  
This pull request includes the verified contract address and logo file for PrideToken (PRD)
  
**Verification Links:**  
- [Etherscan](https://etherscan.io/address/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd#code)  
- [Celo Explorer (Sourcify)](https://sourcify.dev/#/lookup/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd)  
- [BscScan](https://bscscan.com/address/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd#code)  
  
### Token Economics Documentation  
  
Please find the detailed token economics documentation in our GitHub repository: [TOKENOMICS.md](https://github.com/senegalbgc/pride-token-list/blob/main/TOKENOMICS.md)

### CODE REVIEW

**Review by aaronmgdr — CHANGES_REQUESTED** [review]
can you verify the  token contract on celoscan or with sourcify? 

and are there any docs on how the token economics function?

**Comment by senegalbgc** [comment]
Hi @aaronmgdr,  
  
We have updated the pull request with the correct deployment address for the PrideToken (PRD) contract.  
  
**Updated Token Information:**  
- **Name:** PrideToken  
- **Address:** `0x1B0b2647Fc522417d79d8080F9D41977f03D3C46`  
- **Symbol:** PRD  
- **Decimals:** 18  
- **Chain ID:** 42220  
- **Logo URI:** [PrideToken Logo](https://pridefunding.org/assets/PrideToken/PrideTokenbg.svg)  
  
**Verification Details:**  
- **Contract Name:** PrideToken  
- **Network:** Celo Mainnet  
- **Contract Address:** [0x1B0b2647Fc522417d79d8080F9D41977f03D3C46](https://sourcify.dev/#/lookup/0x1B0b2647Fc522417d79d8080F9D41977f03D3C46)  
- **Verification Link:** [Sourcify Verification](https://sourcify.dev/#/lookup/0x1B0b2647Fc522417d79d8080F9D41977f03D3C46)  
  
### Token Economics Documentation  
Please find the detailed token economics documentation in our GitHub repository: [TOKENOMICS.md](https://github.com/senegalbgc/pride-token-list/blob/main/TOKENOMICS.md)  
  
Let us know if you need anything else.

**Comment by aaronmgdr** [comment]
the address in the pr 0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd doesnt match the description or the sourcify link

**Comment by senegalbgc** [comment]
@aaronmgdr here: 

Name: PrideToken
Address: 0x1B0b2647Fc522417d79d8080F9D41977f03D3C46
Symbol: PRD
Decimals: 18
Chain ID: 42220

Logo URI: [PrideToken Logo](https://pridefunding.org/assets/PrideToken/PrideTokenbg.svg)

**Verification Links:**  
- [Etherscan](https://etherscan.io/address/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd#code)  
- [Celo Explorer (Sourcify)](https://sourcify.dev/#/lookup/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd)  
- [BscScan](https://bscscan.com/address/0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd#code) 
- 
Let me know if there is anything else you need.

File: celo.tokenlist.json [hunk 1]
```diff
@@ -624,6 +624,14 @@
...
+      "name": "PrideToken",
+      "address": "0xAE62Ca7e7Bc80C800DADc1c4b9175800033537cd",
+      "symbol": "PRD",
+      "decimals": 18,
+      "chainId": 42220,
+      "logoURI": "https://pridefunding.org/assets/PrideToken/PrideTokenbg.svg"
     }
   ]
-}
+
```
Comment (aaronmgdr) [inline celo.tokenlist.json:637]:
```suggestion
}
```

Comment (aaronmgdr) [inline celo.tokenlist.json:637]:
```suggestion
}
```

Comment (aaronmgdr) [inline celo.tokenlist.json:637]:
```suggestion
}

**Review by aaronmgdr — APPROVED** [review]
🌈

**Comment by senegalbgc** [comment]
Thanks @aaronmgdr!
Natty

**Pull request merged**